Bottom: c82c6f7691faf03cf5555583d1c7095594c6b985
Top:    dabbefd50566c903bf6020f4c6967fb304851054
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-23 23:42:46 +0000

Refresh of fix-labeling-of-contour-which

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 4de4448..0c7b78c 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1222,8 +1222,6 @@ label_contour (PLINE * a)
 
   if (first_labelled == NULL)
     {
-//      g_warning ("Walked entire contour and couldn't find anything we could label - is it INSIDE or OUTSIDE??  SHARED OR SHARED2??");
-
       if (EDGE_LABEL (&a->head) == UNKNWN)
         {
           g_warning ("Walked entire contour and couldn't find anything we could label - it is either all INSIDE or OUTSIDE");
